### PR TITLE
feat: update app database model

### LINKED ISSUE #38: Implement the database model according to ERD from Figma
## Basic Information
ERD Diagram Link - https://cottony-rover-487.notion.site/ERD-4838acae80e34c2ab001e3446956d178

Figma Link - https://www.figma.com/proto/bhSe5HkMJKKN0Bs1ZqSddi/Penni?node-id=5996-22146&t=s4b05Gg69rb6KRmZ-0&scaling=scale-down&content-scaling=fixed&page-id=5996%3A22124&starting-point-node-id=5996%3A23356

### PR DESCRIPTION
**Pull request by 0XFF-96**
## Change Summary
**[Briefly summarise the changes that you made. Just high-level stuff]**

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [x] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**

# Related issue

- Resolve #38

### CODE REVIEW

**Comment by 0XFF-96** [comment]
After migration, the database should have five tables.

**Comment by yunho7687** [comment]
I have updated the main branch. Please update your branch to resolve the flake8 errors.
Please review my commit to see the changes I have made.

File: pyproject.toml
Comment (yunho7687) [inline pyproject.toml:1]: Why there is a `pyproject.toml` file at the root directory?
the dependencies about the backend should be included in the `server` directory right?

Comment (0XFF-96) [reply pyproject.toml:1]: I accidentally upload my local idea config file to github. Will redo the commit, and remove these files. xx

**Review by yunho7687 — APPROVED** [review]
lgtm

**Comment by yunho7687** [comment]
I think you can merge this pr now

**Pull request merged**